tests/unit/Console/SymfonyOutputWriterTest.php

```
@@ -0,0 +1,106 @@
+            "plain <output> fallback\n",
+            $output->fetch(),
+            "When the wrapped 'OutputInterface' is not a 'ConsoleOutputInterface', stderr writes must fall back to "
+            . 'the primary output (still preserving raw bytes).',
+        );
+    }
+
+    public function testWriteStderrRoutesThroughErrorOutputOfConsoleOutputInterface(): void
+    {
+        $console = new ConsoleOutput();
+
+        $errorOutput = $console->getErrorOutput();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$errorOutput'. (undefined)

(UnusedLocalVariable)